packages/whatsie/whatsie.pacscript

```diff
@@ -0,0 +1,19 @@
+name="whatsie"
+gives="whatsie"
+version="4.14.1"
+url="https://github.com/keshavbhatt/whatsie/archive/refs/tags/v${version}.tar.gz"
+hash="369acebb948375713f4e97a5e0304dbda407b432b42fe8ad0b6aa9b602744311"
+homepage="https://github.com/keshavbhatt/whatsie"
+description="Feature rich WhatsApp Client for Desktop Linux"
+maintainer="Diegiwg <diegiwg@gmail.com>"
+depends=("qtbase5-dev" "qtdeclarative5-dev" "qtlocation5-dev" "libqt5webchannel5-dev" "qtwebengine5-dev" "qtwebengine5-dev-tools" "qtwebengine5-private-dev" "libqt5webengine-data")
```

Comment:
These are the runtime dependencies? If yes, I dont think we need *-dev packages. They are only required for building whatsie. Only respective runtime liberaries are required and not the development files (header etc).

Reply:
I tried removing `depends` and it ends up with this error:
```bash
whatsie: error while loading shared libraries: libQt5WebEngineWidgets.so.5: cannot open shared object file: No such file or directory
```

Reply:
On unbutu and debian, all qt5 libs are terminated in *-dev

Reply:
@keshavbhatt does everything look good to you?

Reply:
> I tried removing `depends` and it ends up with this error:
> ```bash
> whatsie: error while loading shared libraries: libQt5WebEngineWidgets.so.5: cannot open shared object file: No such file or directory
> ```

You probably need to declare qt5webenginewidgets5 in runtime dependencies, the shared library resides this package: 
 https://packages.ubuntu.com/kinetic/amd64/libqt5webenginewidgets5/filelist


The *-dev package only contains development header and have no use at runtime. Declaring *-dev packages will work for sure since they also install corresponding shared library packages when pulled. But the ideal way is to not declare them as runtime dependencies. 

Thanks.

Reply:
@Diegiwg can you implement what Keshavbhatt said?

Reply:
Yeap, it finally worked. Thank you so much for your attention, dear ><